shoes-swt/spec/shoes/swt/app_spec.rb

```diff
@@ -122,4 +128,20 @@
       it_behaves_like 'reports client area dimensions'
     end
   end
+
```

Comment:
Please comment below the code you want to discuss (then it shows up on the comments overview :) )